### PR TITLE
Update metadata for Cortex.Cortex version 0.3.2-beta

### PR DESCRIPTION
**Pull request by ghost**
null

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/24815)

### CODE REVIEW

**Review by ItzLevvie — DISMISSED** [review]

File: manifests/c/Cortex/Cortex/0.3.2-beta/Cortex.Cortex.locale.en-US.yaml [hunk 1]
```diff
@@ -7,6 +7,6 @@ PackageLocale: en-US
 Publisher: Cortex
 PackageName: Cortex
 License: Copyright © 2019 Cortex
```
Comment (palenshus) [inline manifests/c/Cortex/Cortex/0.3.2-beta/Cortex.Cortex.locale.en-US.yaml:9]:
```suggestion
PackageUrl: https://crtx.gg/
License: Copyright © 2019 Cortex
```

File: manifests/c/Cortex/Cortex/0.3.2-beta/Cortex.Cortex.installer.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ PackageVersion: 0.3.2-beta
 Installers:
 - Architecture: x64
   InstallerType: nullsoft
-  InstallerUrl: https://download.crtx.gg/Cortex%20Setup%20latest.exe
+  InstallerUrl: https://storage.googleapis.com/crtx-prod.appspot.com/public/releases/Cortex%20Setup%20latest.exe
```
Comment (palenshus) [inline manifests/c/Cortex/Cortex/0.3.2-beta/Cortex.Cortex.installer.yaml:9]: Why this change? At https://docs.crtx.gg/getting-started/installation the download URL still shows as https://download.crtx.gg/Cortex%20Setup%20latest.exe

**Review by palenshus — DISMISSED** [review]

File: manifests/c/Cortex/Cortex/0.3.2-beta/Cortex.Cortex.installer.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/c/Cortex/Cortex/0.3.2-beta/Cortex.Cortex.installer.yaml:9]: URL is a redirect to `https://storage.googleapis.com/crtx-prod.appspot.com/public/releases/Cortex%20Setup%20latest.exe`.

I'm fine with closing the PR if it's not a needed change.

Comment (palenshus) [reply manifests/c/Cortex/Cortex/0.3.2-beta/Cortex.Cortex.installer.yaml:9]: I'd think it'd be better to keep the original URL since we support redirects in the client. If the publisher decides they want to redirect their URL to some other location at some point, this gives them that flexibility.

**Review by ItzLevvie — APPROVED** [review]

**Comment by denelon** [comment]
@ItzLevvie would it help if we added another moderator or two? I don't think we're setting the best example for an individual to submit and approve their own PRs.

**Comment by ItzLevvie** [comment]
Sure, that would be much better. I would vote for @quhxl or @ImJoakim.

**Comment by denelon** [comment]
@ItzLevvie thanks for the nominations. I did a quick look at the "Contributors" data. @quhxl and @ImJoakim both have more than 500 commits each. @ImJoakim has been active for more than the last 90 consecutive days. 

I want to make sure we're keeping the quality bar high. You, @OfficialEsco and @jedieaston all had more than 800 commits and more than six months of consecutive activity. I want to make sure we continue focusing on quality and code of conduct more than just hitting some arbitrary number of PRs and longevity.

I also have concerns about becoming a moderator also turning into a popularity contest. There may be other individuals who are also well qualified to make solid determinations regarding good quality manifests, and supporting the community at large who don't have the "stats" to trivially show up as candidates by numbers alone. That being said, I'd like to discuss with the rest of the team here gather some feedback.

**Comment by denelon** [comment]
@ItzLevvie I spoke with the team. We also recognize @KaranKad as a contributor who has been active. I'd like to draft a specification for what is expected of moderators. I would really appreciate any suggestions you, @OfficialEsco, and @jedieaston have regarding moderation. I'll link the PR back here once I get a draft started. That should hopefully help everyone in the community understand what moderators are doing, and for future moderators what is expected.

**Comment by OfficialEsco** [comment]
What what, didn't get a notification before now, i've been thinking of nominating @quhxl (great PR quality), @ImJoakim (great PR quality and already testing PRs for us) and @KaranKad (Hiding in the dark but helped a lot in the crisis period, less active now that things have calmed down) for a long time but didn't suggest since you said the "800 contribution requirement"

The requirements should not state a contribution requirement, that will just lead to "pointless" PR spam. Which most likely we Moderators will have to go over and suggest changes on...

Currently me and @ItzLevvie are online at the exact same hours, which most of the time leads us to going over PRs at the same time, and @jedieaston are waaaay too busy lately so we don't get much US timezone approvals...

@quhxl and @ImJoakim also live in our TimeZone tho, but it will hopefully relieve me and @ItzLevvie from feeling like having to be on GitHub all day and constantly booting VM/Sandbox 🙄 Not sure about @KaranKad but i think he is US(?)

**Comment by denelon** [comment]
@OfficialEsco I'm not a fan of purely numeric requirements. The intent is to identify individuals who have demonstrated their ability to produce quality manifests (dig into what should be included and shouldn't), honor our code of conduct, and have some longevity. The numbers initially shared were more a reflection of the first three moderators. Numeric requirements tend to drive the behavior you called out with respect to creating a bunch of PRs just for the sake of hitting the numbers.

We also don't want a flood of moderators without having clear guidelines. We want consistency, so the opinions of our current moderators are also highly valued. Sadly, this can lead to a popularity contest, and I'm planning to have some provisions to avoid that as well.

@quhxl, @ImJoakim, and @KaranKad we haven't decided how many moderators to add, but please let us know if any of you are willing to volunteer to help moderate. You may certainly decline, and should feel no pressure to accept the responsibility. The discussion on [moderation](https://github.com/microsoft/winget-pkgs/discussions/15607) should help give you a sense of what we're thinking and what others in the community have said.

**Comment by ImJoakim** [comment]
@denelon I would love to help out moderating to the best of my ability. This is a project that I'm invested in for numerous reasons, and I'd love to have the opportunity to do more for this project.

I appreciate the nominations @ItzLevvie & @OfficialEsco 😃

**Comment by KaranKad** [comment]
I would like to help with moderation.

**Comment by quhxl** [comment]
I am also very happy about the nomination and would love to help!
